nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/scalar/comparison/ScalarLessThan.java

```diff
@@ -65,7 +65,7 @@ public String onnxName() {
 
     @Override
     public String tensorflowName() {
-        return "less";
+        return "LessThan";
```

Comment:
This is still wrong.
Previous comment:
> This mapping is incorrect, and already mapped to LessThan custom op - remove method.

By this I meant that ScalarLessThan should not have a tensorflowName() method at all.
The previous "Less" TF name was incorrect, and was already imported in the ND4J LessThan method: https://github.com/eclipse/deeplearning4j/blob/master/nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/transforms/custom/LessThan.java#L66

So to be clear, delete this tensorflowName() method for the ScalarLessThan op.

Reply:
Done